features/git-town-config/already_set.feature

```diff
@@ -6,14 +6,14 @@ Feature: listing the configuration
 
 
   Background:
-    Given I have branches named "production" and "qa"
+    Given my repository has the branches "production" and "qa"
 
 
   Scenario: everything is configured
-    Given I have configured the main branch name as "main"
-    And my perennial branches are configured as "qa"
+    Given Git Town's main branch is configured as "main"
```

Comment:
Thoughts on `Git Town's` -> `the`. that matches what you did with `hack-push-flag` and could be applied to any other usage of `Git Town's`. The following line would then also need to be updated with `its` -> `the`

Reply:
I like that, its shorter and still gets the message across.